Source/FileSystem/Importers/NavMeshImporter.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+#include "DataModels/Resources/ResourceNavMesh.h"
+#include "FileSystem/Importers/Importer.h"
+
+class NavMeshImporter : public Importer<ResourceNavMesh>
+{
+public:
+	NavMeshImporter(){};
+	~NavMeshImporter(){};
```

Comment:
Missing `override` identifier
```suggestion
	~NavMeshImporter() override {};
```

Reply:
I added it at the beginning and I don't know why I removed it. Nice catch!